package.json

```diff
@@ -85,5 +86,6 @@
     "node-fetch": "^2.6.1",
     "nodemailer": "^6.4.16",
     "nodemailer-sendgrid": "^1.0.3"
-  }
+  },
+  "packageManager": "yarn@1.22.19"
```

Comment:
@gquadrati it's needed on system that uses multiple version of `yarn`, if not specified node picks the latest version of yarn installed on the machine, breaking the project

(it's mandatory for us because some pagopa projects uses `yarn 3`)